test/test_torch.py

```diff
@@ -396,7 +399,8 @@ def cosh(x):
             try:
                 return math.cosh(x)
             except OverflowError:
-                return float('inf') if x > 0 else float('-inf')
+                # http://en.cppreference.com/w/cpp/numeric/math/cosh
```

Comment:
Can you add an extra comment here? It's unclear why should someone go there, and what should they look for.

Reply:
This references shows that for an overflow cosh overflows to inf in any case, no matter the sign of the argument.